docs/nvrm/pmu/ucode-cmds.rst

```diff
@@ -0,0 +1,295 @@
+.. _ucode-cmds:
```

Comment:
These labels are global, you might want to use something more specific

Reply:
Revised version pushed to the branch.